### PR TITLE
Add spec_url for CSS at-rules

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: css/at-rules/font-face.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
         "__compat": {
           "description": "<code>@font-face</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face",
+          "spec_url": "https://drafts.csswg.org/css-fonts-3/#font-face-rule",
```
Comment (Elchi3) [inline css/at-rules/font-face.json:8]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-fonts/#font-face-rule",
```

File: css/at-rules/font-face.json [hunk 2]
```diff
@@ -227,6 +228,7 @@
         "font-display": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-display",
+            "spec_url": "https://drafts.csswg.org/css-fonts-4/#font-display-desc",
```
Comment (Elchi3) [inline css/at-rules/font-face.json:231]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#font-display-desc",
```

File: css/at-rules/font-face.json [hunk 3]
```diff
@@ -275,6 +277,7 @@
         "font-family": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-family",
+            "spec_url": "https://drafts.csswg.org/css-fonts-3/#font-family-desc",
```
Comment (Elchi3) [inline css/at-rules/font-face.json:280]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#font-family-desc",
```

File: css/at-rules/font-face.json [hunk 4]
```diff
@@ -386,6 +389,10 @@
         "font-stretch": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-stretch",
+            "spec_url": [
+              "https://drafts.csswg.org/css-fonts-4/#font-prop-desc",
+              "https://drafts.csswg.org/css-fonts-3/#font-prop-desc"
+            ],
```
Comment (Elchi3) [inline css/at-rules/font-face.json:395]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#descdef-font-face-font-stretch",
```

File: css/at-rules/font-face.json [hunk 5]
```diff
@@ -434,6 +441,10 @@
         "font-style": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-style",
+            "spec_url": [
+              "https://drafts.csswg.org/css-fonts-4/#font-prop-desc",
+              "https://drafts.csswg.org/css-fonts-3/#font-prop-desc"
+            ],
```
Comment (Elchi3) [inline css/at-rules/font-face.json:447]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#descdef-font-face-font-style",
```

File: css/at-rules/font-face.json [hunk 6]
```diff
@@ -482,6 +493,10 @@
         "font-variant": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-variant",
+            "spec_url": [
+              "https://drafts.csswg.org/css-fonts-4/#font-prop-desc",
+              "https://drafts.csswg.org/css-fonts-3/#font-prop-desc"
+            ],
```
Comment (Elchi3) [inline css/at-rules/font-face.json:499]: This entire MDN pages looks wrong to me. There is no `font-variant` descriptor it rather is an ordinary css property?

File: css/at-rules/font-face.json [hunk 7]
```diff
@@ -530,6 +545,7 @@
         "font-variation-settings": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-variation-settings",
+            "spec_url": "https://drafts.csswg.org/css-fonts-4/#font-rend-desc",
```
Comment (Elchi3) [inline css/at-rules/font-face.json:548]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#descdef-font-face-font-variation-settings",
```

File: css/at-rules/font-face.json [hunk 8]
```diff
@@ -604,6 +620,10 @@
         "font-weight": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-weight",
+            "spec_url": [
+              "https://drafts.csswg.org/css-fonts-4/#font-prop-desc",
+              "https://drafts.csswg.org/css-fonts-3/#font-prop-desc"
+            ],
```
Comment (Elchi3) [inline css/at-rules/font-face.json:626]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#descdef-font-face-font-weight",
```

File: css/at-rules/font-face.json [hunk 9]
```diff
@@ -652,6 +672,7 @@
         "src": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/src",
+            "spec_url": "https://drafts.csswg.org/css-fonts-3/#src-desc",
```
Comment (Elchi3) [inline css/at-rules/font-face.json:675]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#src-desc",
```

File: css/at-rules/font-face.json [hunk 10]
```diff
@@ -748,6 +769,7 @@
         "unicode-range": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/unicode-range",
+            "spec_url": "https://drafts.csswg.org/css-fonts-3/#unicode-range-desc",
```
Comment (Elchi3) [inline css/at-rules/font-face.json:772]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-fonts/#unicode-range-desc",
```

File: css/at-rules/font-feature-values.json [hunk 11]
```diff
@@ -5,6 +5,7 @@
         "__compat": {
           "description": "<code>@font-feature-values</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-feature-values",
+          "spec_url": "https://drafts.csswg.org/css-fonts-4/#font-feature-values",
```
Comment (Elchi3) [inline css/at-rules/font-feature-values.json:8]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-fonts/#font-feature-values",
```

File: css/at-rules/import.json [hunk 12]
```diff
@@ -5,6 +5,12 @@
         "__compat": {
           "description": "<code>@import</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@import",
+          "spec_url": [
+            "https://drafts.csswg.org/css-cascade/#at-import",
+            "https://drafts.csswg.org/css-cascade-3/#at-ruledef-import",
+            "https://drafts.csswg.org/mediaqueries-3/#media0",
+            "https://www.w3.org/TR/CSS2/cascade.html#at-import"
+          ],
```
Comment (Elchi3) [inline css/at-rules/import.json:13]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-cascade/#at-import",
```

File: css/at-rules/media.json [hunk 13]
```diff
@@ -5,6 +5,13 @@
         "__compat": {
           "description": "<code>@media</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media",
+          "spec_url": [
+            "https://drafts.csswg.org/mediaqueries-5/#media-descriptor-table",
+            "https://drafts.csswg.org/css-conditional-3/#at-media",
+            "https://drafts.csswg.org/mediaqueries-4/#media",
+            "https://drafts.csswg.org/mediaqueries-3/#media0",
+            "https://www.w3.org/TR/CSS2/media.html#at-media-rule"
+          ],
```
Comment (Elchi3) [inline css/at-rules/media.json:14]:
```suggestion
          "spec_url": [
            "https://drafts.csswg.org/mediaqueries/#media-descriptor-table",
            "https://drafts.csswg.org/css-conditional/#at-media",
          ],
```

File: css/at-rules/media.json [hunk 14]
```diff
@@ -52,6 +59,7 @@
         "any-hover": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/any-hover",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#descdef-media-any-hover",
```
Comment (Elchi3) [inline css/at-rules/media.json:62]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#descdef-media-any-hover",
```

File: css/at-rules/media.json [hunk 15]
```diff
@@ -101,6 +109,7 @@
         "any-pointer": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/any-pointer",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#descdef-media-any-pointer",
```
Comment (Elchi3) [inline css/at-rules/media.json:112]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#descdef-media-any-pointer",
```

File: css/at-rules/media.json [hunk 16]
```diff
@@ -151,6 +160,10 @@
           "__compat": {
             "description": "<code>aspect-ratio</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/aspect-ratio",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#aspect-ratio",
+              "https://drafts.csswg.org/mediaqueries-3/#aspect-ratio"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:166]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#aspect-ratio",
```

File: css/at-rules/media.json [hunk 17]
```diff
@@ -200,6 +213,7 @@
           "__compat": {
             "description": "<code>calc()</code> expressions",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/calc()",
+            "spec_url": "https://drafts.csswg.org/css-values-3/#calc-notation",
```
Comment (Elchi3) [inline css/at-rules/media.json:216]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-values/#calc-notation",
```

File: css/at-rules/media.json [hunk 18]
```diff
@@ -249,6 +263,10 @@
           "__compat": {
             "description": "<code>color</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/color",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#color",
+              "https://drafts.csswg.org/mediaqueries-3/#color"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:269]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#color",
```

File: css/at-rules/media.json [hunk 19]
```diff
@@ -298,6 +316,7 @@
           "__compat": {
             "description": "<code>color-gamut</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/color-gamut",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#color-gamut",
```
Comment (Elchi3) [inline css/at-rules/media.json:319]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#color-gamut",
```

File: css/at-rules/media.json [hunk 20]
```diff
@@ -347,6 +366,10 @@
           "__compat": {
             "description": "<code>color-index</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/color-index",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#color-index",
+              "https://drafts.csswg.org/mediaqueries-3/#color-index"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:372]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#color-index",
```

File: css/at-rules/media.json [hunk 21]
```diff
@@ -396,6 +419,10 @@
           "__compat": {
             "description": "<code>device-aspect-ratio</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/device-aspect-ratio",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#device-aspect-ratio",
+              "https://drafts.csswg.org/mediaqueries-3/#device-aspect-ratio"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:425]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#device-aspect-ratio",
```

File: css/at-rules/media.json [hunk 22]
```diff
@@ -445,6 +472,10 @@
           "__compat": {
             "description": "<code>device-height</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/device-height",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#device-height",
+              "https://drafts.csswg.org/mediaqueries-3/#device-height"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:478]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#device-height",
```

File: css/at-rules/media.json [hunk 23]
```diff
@@ -494,6 +525,10 @@
           "__compat": {
             "description": "<code>device-width</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/device-width",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#device-width",
+              "https://drafts.csswg.org/mediaqueries-3/#device-width"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:531]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#device-width",
```

File: css/at-rules/media.json [hunk 24]
```diff
@@ -594,6 +630,7 @@
           "__compat": {
             "description": "<code>forced-colors</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/forced-colors",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-5/#forced-colors",
```
Comment (Elchi3) [inline css/at-rules/media.json:633]: Here `-5` is actually correct.

File: css/at-rules/media.json [hunk 25]
```diff
@@ -661,6 +698,10 @@
           "__compat": {
             "description": "<code>grid</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/grid",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#grid",
+              "https://drafts.csswg.org/mediaqueries-3/#grid"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:704]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#grid",
```

File: css/at-rules/media.json [hunk 26]
```diff
@@ -710,6 +751,10 @@
           "__compat": {
             "description": "<code>height</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/height",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#height",
+              "https://drafts.csswg.org/mediaqueries-3/#height"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:757]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#height",
```

File: css/at-rules/media.json [hunk 27]
```diff
@@ -758,6 +803,7 @@
         "hover": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/hover",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#hover",
```
Comment (Elchi3) [inline css/at-rules/media.json:806]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#hover",
```

File: css/at-rules/media.json [hunk 28]
```diff
@@ -955,6 +1002,10 @@
           "__compat": {
             "description": "<code>monochrome</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/monochrome",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#monochrome",
+              "https://drafts.csswg.org/mediaqueries-3/#monochrome"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:1008]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#monochrome",
```

File: css/at-rules/media.json [hunk 29]
```diff
@@ -1052,6 +1103,10 @@
           "__compat": {
             "description": "<code>orientation</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/orientation",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#orientation",
+              "https://drafts.csswg.org/mediaqueries-3/#orientation"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:1109]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#orientation",
```

File: css/at-rules/media.json [hunk 30]
```diff
@@ -1101,6 +1156,7 @@
           "__compat": {
             "description": "<code>overflow-block</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/overflow-block",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#mf-overflow-block",
```
Comment (Elchi3) [inline css/at-rules/media.json:1159]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#mf-overflow-block",
```

File: css/at-rules/media.json [hunk 31]
```diff
@@ -1150,6 +1206,7 @@
           "__compat": {
             "description": "<code>overflow-inline</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/overflow-inline",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#mf-overflow-inline",
```
Comment (Elchi3) [inline css/at-rules/media.json:1209]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#mf-overflow-inline",
```

File: css/at-rules/media.json [hunk 32]
```diff
@@ -1198,6 +1255,7 @@
         "pointer": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/pointer",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#pointer",
```
Comment (Elchi3) [inline css/at-rules/media.json:1258]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#pointer",
```

File: css/at-rules/media.json [hunk 33]
```diff
@@ -1631,6 +1694,7 @@
           "__compat": {
             "description": "<code>resolution</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/resolution",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-3/#resolution",
```
Comment (Elchi3) [inline css/at-rules/media.json:1697]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#resolution",
```

File: css/at-rules/media.json [hunk 34]
```diff
@@ -1708,6 +1772,10 @@
           "__compat": {
             "description": "<code>scan</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/scan",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#scan",
+              "https://drafts.csswg.org/mediaqueries-3/#scan"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:1778]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#scan",
```

File: css/at-rules/media.json [hunk 35]
```diff
@@ -1861,6 +1930,7 @@
           "__compat": {
             "description": "<code>update</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/update-frequency",
+            "spec_url": "https://drafts.csswg.org/mediaqueries-4/#update",
```
Comment (Elchi3) [inline css/at-rules/media.json:1933]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#update",
```

File: css/at-rules/media.json [hunk 36]
```diff
@@ -1910,6 +1980,10 @@
           "__compat": {
             "description": "<code>width</code> media feature",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/width",
+            "spec_url": [
+              "https://drafts.csswg.org/mediaqueries-4/#width",
+              "https://drafts.csswg.org/mediaqueries-3/#width"
+            ],
```
Comment (Elchi3) [inline css/at-rules/media.json:1986]:
```suggestion
            "spec_url": "https://drafts.csswg.org/mediaqueries/#width",
```

File: css/at-rules/namespace.json [hunk 37]
```diff
@@ -5,6 +5,7 @@
         "__compat": {
           "description": "<code>@namespace</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@namespace",
+          "spec_url": "https://drafts.csswg.org/css-namespaces-3/#declaration",
```
Comment (Elchi3) [inline css/at-rules/namespace.json:8]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-namespaces/#declaration",
```

File: css/at-rules/page.json [hunk 38]
```diff
@@ -5,6 +5,11 @@
         "__compat": {
           "description": "<code>@page</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page",
+          "spec_url": [
+            "https://drafts.csswg.org/css-logical/#page",
+            "https://drafts.csswg.org/css-page-3/#at-page-rule",
+            "https://www.w3.org/TR/CSS2/page.html#page-selectors"
+          ],
```
Comment (Elchi3) [inline css/at-rules/page.json:12]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-logical/#page",
```

File: css/at-rules/page.json [hunk 39]
```diff
@@ -53,6 +58,7 @@
           "__compat": {
             "description": "<code>bleed</code> descriptor",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/bleed",
+            "spec_url": "https://drafts.csswg.org/css-page-3/#bleed",
```
Comment (Elchi3) [inline css/at-rules/page.json:61]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-page/#bleed",
```

File: css/at-rules/page.json [hunk 40]
```diff
@@ -102,6 +108,7 @@
           "__compat": {
             "description": "<code>marks</code> descriptor",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/marks",
+            "spec_url": "https://drafts.csswg.org/css-page-3/#marks",
```
Comment (Elchi3) [inline css/at-rules/page.json:111]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-page/#marks",
```

File: css/at-rules/page.json [hunk 41]
```diff
@@ -249,6 +256,7 @@
           "__compat": {
             "description": "<code>size</code> descriptor",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/size",
+            "spec_url": "https://drafts.csswg.org/css-page-3/#page-size-prop",
```
Comment (Elchi3) [inline css/at-rules/page.json:259]:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-page/#page-size-prop",
```

File: css/at-rules/page.json [hunk 42]
```diff
@@ -297,6 +305,7 @@
             "__compat": {
               "description": "<code>jis-b4</code> page size",
               "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/size",
+              "spec_url": "https://drafts.csswg.org/css-page-3/#page-size-prop",
```
Comment (Elchi3) [inline css/at-rules/page.json:308]:
```suggestion
              "spec_url": "https://drafts.csswg.org/css-page/#page-size-prop",
```

File: css/at-rules/page.json [hunk 43]
```diff
@@ -346,6 +355,7 @@
             "__compat": {
               "description": "<code>jis-b5</code> page size",
               "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/size",
+              "spec_url": "https://drafts.csswg.org/css-page-3/#page-size-prop",
```
Comment (Elchi3) [inline css/at-rules/page.json:358]:
```suggestion
              "spec_url": "https://drafts.csswg.org/css-page/#page-size-prop",
```

File: css/at-rules/supports.json [hunk 44]
```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>@supports</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@supports",
+          "spec_url": [
+            "https://drafts.csswg.org/css-conditional-4/#at-supports-ext",
+            "https://drafts.csswg.org/css-conditional-3/#at-supports"
+          ],
```
Comment (Elchi3) [inline css/at-rules/supports.json:11]:
```suggestion
          "spec_url": [
            "https://drafts.csswg.org/css-conditional-4/#at-supports-ext",
            "https://drafts.csswg.org/css-conditional/#at-supports"
          ],
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Thanks! 
I have lots of suggestions here, given I don't like linking to outdated specs. If there is a good reason to have them, I'd like to know why.

File: css/at-rules/font-face.json (hunk 6 above)
Comment (sideshowbarker) [reply css/at-rules/font-face.json:493]:
```
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-variant",
```

> This entire MDN pages looks wrong to me. There is no `font-variant` descriptor it rather is an ordinary css property?

No idea… Maybe @rachelandrew has some guidance

Comment (rachelandrew) [reply css/at-rules/font-face.json:493]: I think it's the same as the others that are listed under `@font-face` https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face in terms of where it is in the docs.

Comment (Elchi3) [reply css/at-rules/font-face.json:493]: Thanks for looking into this! As far as I see it, the other pages under `@font-face` are actual `@font-face` descriptors, but I don't think `font-variant` is a descriptor of `@font-face`, or is it? At least, it isn't listed here: https://drafts.csswg.org/css-fonts/#font-face-descriptor-table

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-variant
https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant

**Review by Elchi3 — APPROVED** [review]
Let's merge this.

I'll ask about https://developer.mozilla.org/docs/Web/CSS/@font-face/font-variant separately.

**Pull request merged**